java/mykola-melnyk/src/com/zoolatech/lecture6/tasks/_3/UserService.java

```diff
@@ -0,0 +1,20 @@
+package com.zoolatech.lecture6.tasks._3;
+
+public class UserService {
+    UserRepository userRepoInst;
```

Comment:
`userRepoInst` should be defined as private

Reply:
Done.